src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -667,6 +676,56 @@ public float AudioPitch
             }
         }
 
+        /// <summary>
+        /// Gets or sets the default codec type of the audio decoder.
+        /// </summary>
+        /// <value>A <see cref="CodecType"/> that specifies the type.
```

Comment:
It is a noun clause. The clause is awkward.

How about writing a sentence?
/// <value>A <see cref="CodecType"/> specifies the type.